Bottom: 04f4dddaa30295126675db3dce56b71cb3bccc02
Top:    354dbb14fd2be465d26fc0e496bf084453d1560f
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-01-21 21:29:37 +0000

Refresh of hid-gtk-remove-xor-colour-logi

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 71754a5..45b9b72 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -471,6 +471,16 @@ ghid_set_line_width (hidGC gc, Coord width)
   gc->width = width;
 }
 
+
+void
+ghid_set_draw_xor (hidGC gc, int xor)
+{
+  /* NOT IMPLEMENTED */
+
+  /* Only presently called when setting up a crosshair GC.
+   * We manage our own drawing model for that anyway. */
+}
+
 void
 ghid_set_draw_faded (hidGC gc, int faded)
 {
